src/interface.ts

```diff
@@ -58,6 +59,11 @@ export interface INetworkProvider {
      */
     getNonFungibleTokenOfAccount(address: IAddress, collection: string, nonce: number): Promise<NonFungibleTokenOfAccountOnNetwork>;
 
+    /**
+     * Fetches data about available mex pairs.
+     */
+    getMexPairs(pagination?: IPagination): Promise<PairOnNetwork[]>;
```

Comment:
Perhaps remove this from the interface? If one needs it, one should reference the concrete class, and add interfaces locally.

That is, with respect to:

> The default classes from @elrondnetwork/erdjs-network-providers should only be used as a starting point. As your dApp matures, make sure you switch to using your own network provider, tailored to your requirements (whether deriving from the default ones or writing a new one, from scratch) that directly interacts with the Elrond API (or Gateway).

Extracted from: https://docs.elrond.com/sdk-and-tools/erdjs/extending-erdjs/#extending-the-network-providers

Reply:
done